README.md

```diff
@@ -99,10 +99,10 @@ The following instructions will guide you through the process of setting up Hone
 Use the following examples as a reference for how to run `Honeyscanner`:
 
 ```bash
-python3 main.py --honeypot cowrie --honeypot_version 2.5.0 --target_ip 127.0.0.1 --port 2222 --username root --password 1234
+sudo python3 main.py --honeypot cowrie --honeypot_version 2.5.0 --target_ip 127.0.0.1 --port 2222 --username root --password 1234
```

Comment:
Let's avoid introducing running honeyscanner with `root` privileges, we are just scanning for open ports, so we can change the nmap flags to only '-p-', this way we can keep running honeyscanner as a normal user.